src/platforms/esp32/test/main/test_erl_sources/test_crypto.erl

```diff
@@ -72,6 +72,45 @@ test_hash() ->
     ok = expect(badarg, fun() -> crypto:hash(sha, ["not", "a", "byte", 32768]) end),
     ok = expect(badarg, fun() -> crypto:hash(sha, ["not", "a", "byte", 34359738367]) end),
 
+    <<50, 136, 204, 108, 55>> = crypto:crypto_one_time(
```

Comment:
This should be in a separate function called `test_crypto_one_time` or something

Reply:
Fixed.